Doc/library/typing.rst

```diff
@@ -2122,7 +2313,7 @@ Abstract Base Classes
 Corresponding to collections in :mod:`collections.abc`
 """"""""""""""""""""""""""""""""""""""""""""""""""""""
 
-.. class:: AbstractSet(Collection[T_co])
+.. class:: AbstractSet[T_co](Collection[T_co])
```

Comment:
Do we want to do a purge of all the Hungarian notation in typing.rst, on the grounds that it's annoyingly verbose, and users should no longer have to worry about whether a TypeVar is covariant or invariant?

Reply:
My thinking was that users of these collections still need to care, because it affects how you can use them.

Then again, none of typing.rst explains anything about variance, so I don't know how useful it is.

Reply:
Yeah, I can see both POVs. Another point in favour of maybe keeping the Hungarian notation for now: it'll be a long time before 3.11 goes EOL, and we'll likely see widespread usage of old-style TypeVars until we get to that point. Another point in favour of ditching the Hungarian notation: it's perhaps a bit strange for us to be meticulous about this in `typing.rst`, considering we just dropped all Hungarian notation in `typing.py` in https://github.com/python/cpython/commit/0cb2fdc6217aa7c04b5c798cfd195c8d0f4af353

Reply:
Butting in without full context, but I think it still makes sense to _somehow_ document the variance of type vars for cases where it's not all covariant (covariant is what users who don't think hard about it expect). But I hope that the Hungarian notation will be eradicated.

Reply:
I think I'd like to leave this as is for now, and perhaps in a separate PR we can clarify what "T_co" means, as there is currently nothing in typing.rst that explains it.